README.md

```diff
@@ -24,6 +24,10 @@ and open your browser at `http://localhost:8080/docs/` to see the docs.
 
 To run the server on a Docker container, please execute the following from the root directory:
 
+```bash
+echo "AWS_ACCESS_KEY_ID=foo\nAWS_SECRET_ACCESS_KEY=bar\nAWS_DEFAULT_REGION=us-east-1\nAWS_DYNAMODB_HOST=http://host.docker.internal:4566" > localstack.env
```

Comment:
you can also add localstack.env in the repository.

Reply:
I had the impression that .env files should generally stay out of repositories. I can remove it from the .gitignore then.

Reply:
It's okay as it only has dummy values. We should not add real secrets but this env file would help setup by reducing the step.